bazel/repositories.bzl

```diff
@@ -229,3 +229,17 @@ def proxy_wasm_cpp_host_repositories():
         patches = ["@proxy_wasm_cpp_host//bazel/external:llvm.patch"],
         patch_args = ["-p1"],
     )
+
```

Comment:
Nit: Please move this above Wasmtime and add a separator, since right now this is listed under WAVM.

Reply:
Fixed.